asv/util.py

```diff
@@ -819,8 +818,7 @@ def iter_subclasses(cls):
     """
     for x in cls.__subclasses__():
         yield x
-        for y in iter_subclasses(x):
-            yield y
+        yield from iter_subclasses(x)
```

Comment:
Is there a way to leave the explicit loop?

Reply:
I can add `noqa: B028` as well.

But what's wrong with `yield from`?